workspace/data-sources/data.pages_404.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
Pumping the 403 and 404 from the page section is not a good idea. Please remove this.